searx/engines/google_videos.py

```diff
@@ -17,83 +61,152 @@
     "use_official_api": False,
     "require_api_key": False,
     "results": 'HTML',
+    "template": 'video.html',
+    "parse": ('url', 'title', 'content', 'thumbnail')
 }
 
+logger = logger.getChild('google video')
+
 # engine dependent config
+
 categories = ['videos']
-paging = True
-safesearch = True
+paging = False
+language_support = True
+use_locale_domain = True
 time_range_support = True
-number_of_results = 10
+safesearch = True
+language_aliases = {}
```

Comment:
Why is it necessary ?

According to 
https://github.com/searx/searx/blob/f4a17acb7a4fa82c5cb629f4eaad11ef528f89e4/searx/engines/__init__.py#L117-L120

`language_aliases` is initialized to `{}` even without the line 78, but I most probably miss something here.

Reply:
The languages implementations are hard to understand for me .. but with your hint I looked again and see, it is not needed.

For me it is not clear for what language_aliases is good and when/why it is used or set ..

https://github.com/searx/searx/blob/f4a17acb7a4fa82c5cb629f4eaad11ef528f89e4/searx/engines/__init__.py#L122-L128